### PR TITLE
Fix typos in 2016-03

### PR DESCRIPTION
**Pull request by azu**
Fix some typos and add/fix link to slide.

/cc @bterlson @rwaldron

### CODE REVIEW

File: es7/2016-03/march-30.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ LB: to work out details on a github issue.
 
 ## Duplicate sloppy mode function in block web compatibility fix
 
-Shu-yu Guo and Daniel Ehrenberg presenting slides: https://docs.google.com/presentation/d/1E LM_X4_EMEhcmfiA_AHJ4zw0aKDM8DBxqFOmNsiMHig/edit#slide=id.p
+Shu-yu Guo and Daniel Ehrenberg presenting slides: https://docs.google.com/presentation/d/1E_LM_X4_EMEhcmfiA_AHJ4zw0aKDM8DBxqFOmNsiMHig/edit#slide=id.p
```
Comment (azu) [inline es7/2016-03/march-30.md]: `1E LM_X4` -> `1E_LM_X4`
Previously, link was 404.

File: es7/2016-03/march-30.md [hunk 2]
```diff
@@ -549,7 +549,7 @@ MM: They could also create a replacer
 
 ## Draft Proposed Frozen Realm API
 
-Chip Morningstar presenting slides (TODO: CM, please add slides)
+Chip Morningstar presenting slides https://github.com/FUDCo/proposal-frozen-realms/blob/master/FrozenRealmsPresentation-2016-03-30.pdf
```
Comment (azu) [inline es7/2016-03/march-30.md]: Add link to presentation.

**Comment by azu** [comment]
ping @rwaldron ?

**Comment by azu** [comment]
Update: add commit  e61fbfb

**Comment by azu** [comment]
@rwaldron How are you?

**Comment by julmot** [comment]
@azu One question: Why didn't you just fixed those typos in one commit?

**Comment by azu** [comment]
@julmot If this pull request contain wrong commit, it is easy to revert a commit.
Currently, Repository owner can [squash and merge](https://github.com/blog/2141-squash-your-commits) on GitHub.
So, I think that separated commits has a benefit.

**Comment by azu** [comment]
@rwaldron What is the status of this and #42 ?

**Comment by rwaldron** [comment]
Sorry for the delay in landing this—thanks again (for being patient and for the work)

**Comment by azu** [comment]
@rwaldron No problem! Thanks.

**Pull request merged**